### DISCUSSION TITLE
用ubuntu 跑 xsim 遇到bug會直接閃退

### DISCUSSION THREAD

**Discussion by DonghanTsai** [bol-edu/HLS-SOC-Discussions#205] [General]
我在ubuntu上跑xsim時，
發現有時候跑下去整個terminal會直接消失
測試後發現只要整個程式裡面有一點點bug (ex. 分號忘記打)
它就會不告訴我bug出在哪一行而是直接停止模擬然後關閉terminal視窗
但之前不會這樣的
請問要怎麼修復這個情況

**Comment by jxes993409** [comment]
不要用 source ./run_sim
改用 ./run_sim 就可以了
